bip-0174.mediawiki

```diff
@@ -729,6 +729,8 @@ If an updater is updating a PSBT and needs to add a field that is only available
 New fields should first be proposed on the bitcoin-dev mailing list.
 If a field requires significatn description as to its usage, it should be accompanied by a separate BIP.
 The field must be added to the field listing tables in the Specification section.
+New fields with type 0xFD or higher must be added to PSBT version 2 or newer,
```

Comment:
I don't think this restriction is necessary. Old parsers will regard such fields as unknown anyways, and with the length before the type, no data will be missed.

Reply:
I rephrased it to explain why it's safe.